libs/shared/data-access-strapi/src/lib/page.service.ts

```diff
@@ -83,6 +83,14 @@ export class BaPageService<T = any> {
     return of(this._cache.get(key)!);
   }
 
+  _getCategories(): Observable<string[]> {
```

Comment:
```suggestion
  getCategories(): Observable<string[]> {
```
This is the public API of the service, we should not need a prefix here.